docs-internals/contributions.md

```diff
@@ -5,5 +5,5 @@ Unfortunately outside contributor PRs can't run our tests due to our GHA never s
 
 As a workaround, we can create a new temporary PR off of the PR.  
 First, review the PR change is satisfactory and ensure it is safe to run the tests for (no malicious attempt to steal secrets).  
-Then apply the label [approved-for-tests](https://github.com/earthly/earthly/labels/approved-for-tests). This would trigger a GHA that would open a new draft PR, linked to the original.  
+Then apply the label [approved-for-tests](https://github.com/earthbuild/earthbuild/labels/approved-for-tests). This would trigger a GHA that would open a new draft PR, linked to the original.  
```

Comment:
I don't think this applies anymore. perhaps it should?

I don't have experience with how to securely set up CI in an open source project.